src/ToDoList/ToDoList.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { ToDo } from '../Todo/Todo';
+import './ToDoList.scss';
+import { template } from '../objFit';
+
+export const ToDoList = ({ toDos }) => (
+  <ul>
+    {toDos.map(item => (
+      <li className="ToDo" key={item.id}><ToDo toDo={item} /></li>
```

Comment:
Відформатуй код, зроби його читабельнішим
```suggestion
      <li className="ToDo" key={item.id}>
        <ToDo toDo={item} />
      </li>
```